scripts/data/supersearch_fields.json

```diff
@@ -1207,26 +1195,6 @@
         "form_field_choices": null,
         "name": "startup_time"
     },
-    "dump": {
-        "default_value": null,
-        "is_mandatory": false,
-        "form_field_type": "StringField",
-        "is_returned": false,
-        "query_type": "string",
-        "permissions_needed": [],
-        "storage_mapping": {
-            "index": "not_analyzed",
-            "type": "string"
-        },
-        "namespace": "processed_crash",
-        "name": "dump",
-        "has_full_version": false,
-        "data_validation_type": "str",
-        "in_database_name": "dump",
-        "is_exposed": false,
-        "form_field_choices": [],
-        "description": ""
-    },
```

Comment:
That field doesn't exist anymore. It was a bit stupid to remove it there, it's out of scope, sorry about that.